overcast/django/apps/buildsvc/pkgbuild/__init__.py

```diff
@@ -13,12 +14,14 @@
 from ..models import BuildRecord
 
 class PackageBuilder(object):
-    def __init__(self, basedir, package_source, build_record):
+    def __init__(self, basedir, package_source, build_record,
+                 docker_build=True):
```

Comment:
I thought if one need local sbuild based builds, can pass it to PackageBuilder instance. Isnt that the correct way?